docs/workflows/api-token.md

```diff
@@ -301,13 +331,13 @@ fmt.Printf("%s\n", bodyText)
 </TabItem>
 </Tabs>
 
-### Revoking Access Tokens:
+## Revoking Access Tokens:
```

Comment:
Nit: I'm curious. What's the intended Table of Contents? The default behavior is to display all h2 and h3 headings, so we could remove this if it was intended as an h3, unless it is now intended as an h2.
```
# Display h2 to h2 headings
toc_min_heading_level: 2
toc_max_heading_level: 2
```

Reply:
It is intended for h2 now. Does that mean we should keep this? I wasn't actually sure what this code was doing!

Reply:
Yes, this is fine 🙂 This code overrides the default (h2 & h3) to only show h2 headers in the table of contents. In my other branch, I've actually removed this.

Reply:
oh I see! I removed this too now.